tooling/Microsoft.VisualStudio.Mac.RazorAddin/Properties/_Manifest.addin.xml

```diff
@@ -10,8 +10,8 @@
     <Import assembly="Microsoft.VisualStudio.Mac.LanguageServices.Razor.dll" />
   </Runtime>
   <Dependencies>
-    <Addin id="::MonoDevelop.Core" version="7.4" />
-    <Addin id="::MonoDevelop.Ide" version="7.4" />
+    <Addin id="::MonoDevelop.Core" version="7.5" />
```

Comment:
These versions grow with the version of VS for mac

Reply:
Kinda sucks that we have to do this in 3 different places.

Reply:
Totally agree. The one in C# is totally a convenience piece. Eventually we could generate this manifest.addin.xml based off of msbuild if we find ourselves touching all this a lot.